tests/ft_mcharsetdetector/ft_mcharsetdetector.cpp

```diff
@@ -194,6 +194,10 @@ void Ft_MCharsetDetector::testConstructors()
                 << " errorString: " << charsetDetector1.errorString() << "\n";
 #endif
 
+    if (charsetName.contains("EUC-JP")) {
+        QEXPECT_FAIL("","This test if buggy between ICU releases", Abort);
+    }
+    QCOMPARE(result1, textResult);
```

Comment:
Twice this check? Besides the expected failure test is "Invalid EUC-JP, it is really Latin1" and has some differentiation between qt versions. Could perhaps just remove the row.